css-overflow-3/Overview.bs

```diff
@@ -571,11 +571,13 @@ Scrollbars and Layout</h3>
 <h3 id="scrolling-direction">
 Scrolling Origin, Direction, and Restriction</h3>
 
-	The <dfn export>initial scroll position</dfn>,
-	that is, the initial position of
+	The <dfn export>initial scroll position</dfn> is
+	the initial position of
 	the box’s <a>scrollable overflow area</a>
 	with respect to its border box,
-	prior to any user or programmatic scrolling that changes it,
+	prior to any user or programmatic scrolling that changes it.
+
+	The <a>initial scroll position</a>
```

Comment:
Run on sentence. Broken up into a definition and then another sentence on writing mode.